src/locale/phrases/phrases-en.js

```diff
@@ -309,7 +309,7 @@ const phrases = {
       PUBLISHED: 'Learningpath will be accesible by search and url.',
       PRIVATE: 'Only you can see the learningpath.',
       SUBMITTED:
-        'Learningpath will be accesible by search and url after approval by an admin.',
+        'Learningpath will be accesible by search and url.',
```

Comment:
SUBMITTED vil ikkje være søkbar. Så korrekt tekst er nok heller:

```suggestion
        'Learningpath will be accessible by url, and in search when approved.',
```